lms/templates/preview_menu.html

```diff
@@ -52,7 +52,7 @@
             )
     %>
     <nav class="wrapper-preview-menu" aria-label="${_('Course View')}">
-        <div class="preview-menu" style="display: flex; align-items: center;">
+        <div class="preview-menu" style="display: flex; align-items: center; flex-wrap: wrap; justify-content: space-around;">
```

Comment:
@aayushagra, instead of using inline styles, this could have been added in an external stylesheet. But we already have some inline styles for this element, so 🤷🏼